src/utils.ts

```diff
@@ -0,0 +1,66 @@
+import { randomBytes } from "crypto";
```

Comment:
I think we removed dependency on `crypto` package in https://github.com/stellar/js-stellar-base/pull/169. You should probably use `randombytes`.

Reply:
@bartekn thanks for the heads up, fixed!